data/mods/Aftershock/items/comestibles/alienfood.json

```
@@ -132,5 +173,406 @@
+    "flags": [ "EATEN_HOT", "NUTRIENT_OVERRIDE" ]
+  },
+  {
+    "id": "alien_meat_scrap",
+    "copy-from": "meat",
+    "type": "COMESTIBLE",
+    "name": { "str": "scrap of alien flesh", "str_pl": "scraps of alien flesh" },
+    "description": "This is a tiny scrap of edible alien meat meat.  It's not much, but it might do in a pinch.",
+    "cooks_like": "meat_scrap_cooked",
+    "proportional": { "weight": 0.1, "volume": 0.1, "price": 0.1, "calories": 0.1 },
+    "delete": { "flags": [ "SMOKABLE" ] },
+    "vitamins": [ [ "vitA", 0 ], [ "vitC", 0 ], [ "calcium", 0 ], [ "iron", 0 ], [ "vitB", 0 ], [ "mutant_toxin", 5 ] ]
+  },
+  {
+    "id": "alien_meat_scrap_cooked",
+    "copy-from": "alien_meat_scrap",
+    "type": "COMESTIBLE",
+    "name": { "str": "cooked scrap of alien flesh", "str_pl": "cooked scraps of alien flesh" },
+    "parasites": 0,
+    "fun": 0,
+    "healthy": 0,
+    "flags": [ "NUTRIENT_OVERRIDE" ],
+    "calories": 51
+  },
+  {
+    "id": "alien_stomach",
+    "copy-from": "flesh",
+    "type": "COMESTIBLE",
+    "name": "alien stomach",
+    "description": "A balloon like organ you can only guess is a stomach.  It is surprisingly durable.",
+    "weight": "72 g",
+    "volume": "250 ml",
+    "price_postapoc": 25,
+    "price": 200,
+    "spoils_in": "8 hours",
+    "calories": 60,
+    "quench": 1,
+    "healthy": -1,
+    "fun": -15,
+    "vitamins": [ [ "mutant_toxin", 25 ] ],
+    "extend": { "flags": [ "PREDATOR_FUN" ] }
+  },
+  {
+    "id": "alien_stomach_large",
+    "copy-from": "alien_stomach",
+    "type": "COMESTIBLE",
+    "name": "large alien stomach",
+    "description": "A balloon like organ of a large alien.  It is surprisingly durable.",
+    "proportional": { "weight": 2.0, "volume": 2.0, "price": 1.5, "calories": 2.0 }
+  },
+  {
+    "id": "alien_lung",
+    "type": "COMESTIBLE",
+    "copy-from": "flesh",
+    "looks_like": "offal",
+    "name": { "str": "piece of raw alien lung", "str_pl": "pieces of raw alien lung" },
+    "description": "A red sponge-like organ rests in your hands.  It leaks out a strange liquid, and spoils very quickly.  You hazard a guess this is a lung.",
+    "weight": "56 g",
+    "volume": "250 ml",
+    "color": "pink",
+    "charges": 4,
+    "spoils_in": "1 day",
+    "healthy": -2,
+    "price_postapoc": 25,
+    "quench": 2,
+    "fun": -10,
+    "calories": 50,
+    "vitamins": [ [ "vitA", 0 ], [ "vitC", 9 ], [ "calcium", 0 ], [ "iron", 14 ], [ "vitB", 22 ], [ "mutant_toxin", 25 ] ],
+    "extend": { "flags": [ "PREDATOR_FUN" ] }
+  },
+  {
+    "id": "alien_lung_cooked",
+    "type": "COMESTIBLE",
+    "copy-from": "alien_lung",
+    "looks_like": "offal_cooked",
+    "name": { "str": "cooked piece of alien lung", "str_pl": "cooked pieces of alien lung" },
+    "description": "The organ no longer leaks out a strange liquid, and has faded to a light blue color. It is chewy, bitter, and bad-smelling.",
```

```suggestion
    "description": "The organ no longer leaks out a strange liquid, and has faded to a light blue color. It looks chewy, and has an unpleasant smell.",
```
Thought it would be better to describe it as if it were still untasted